### DISCUSSION TITLE
Image syntax and link creation

### DISCUSSION THREAD

**Discussion by avoidaccount** [jakewvincent/mkdnflow.nvim#215] [Q&A] [answered]
It is possible to make mkdnflow ignore the exclamation on the image syntax `![alt](path)`?
I noted that if Enter or FollowLink is used on the alt it opens the image as expected, but the **!** created a new link instead.
Im using the default config.

**Comment by jakewvincent** [comment, marked as answer]
Hi @avoidaccount, I'm going to create an issue from this. This is a relevant request, and I think this should be built in.
